lib/license_scout/overrides.rb

```diff
@@ -373,6 +373,7 @@ def default_overrides
         ["googleapis-common-protos", "Apache-2.0", ["https://raw.githubusercontent.com/googleapis/googleapis/master/LICENSE"]],
         ["grpc-google-iam-v1", "Apache-2.0", ["https://raw.githubusercontent.com/googleapis/googleapis/master/LICENSE"]],
         ["grpc", "Apache-2.0", ["https://raw.githubusercontent.com/grpc/grpc/master/LICENSE"]],
+        ["get_process_mem", "MIT", ["LICENSE"]],
```

Comment:
I don's see the LICENSE file at https://github.com/schneems/get_process_mem should it be : https://raw.githubusercontent.com/schneems/get_process_mem/master/README.md